scripts/import/import_EPMC.pl

```diff
@@ -744,7 +744,7 @@ sub clean_publications{
 
     # If there is publications without title, delete them 
     if($n_title_null != 0){
-      my $title_null_sth = $dba->dbc->prepare(qq[ select publication_id from publication where title is null or title = '' ]);
+      my $title_null_sth = $dba->dbc->prepare(qq[ select publication_id, pmid from publication where title is null or title = '' ]);
```

Comment:
Where is the added pmid used?

Reply:
When removes the publication it write in a file the publication_id and in this case the tile is null so can't write the title, write the pmid instead. Writing only the publication_id is not info enough.